Source/WebKit/WebProcess/cocoa/PlaybackSessionManager.mm

```diff
@@ -217,35 +217,36 @@
     return std::make_tuple(WTFMove(model), WTFMove(interface));
 }
 
-PlaybackSessionManager::ModelInterfaceTuple& PlaybackSessionManager::ensureModelAndInterface(PlaybackSessionContextIdentifier contextId)
+const PlaybackSessionManager::ModelInterfaceTuple& PlaybackSessionManager::ensureModelAndInterface(PlaybackSessionContextIdentifier contextId)
 {
-    auto addResult = m_contextMap.add(contextId, ModelInterfaceTuple());
-    if (addResult.isNewEntry)
-        addResult.iterator->value = createModelAndInterface(contextId);
+    auto it = m_contextMap.find(contextId);
+    if (it != m_contextMap.end())
+        return it->value;
+    auto addResult = m_contextMap.add(contextId, createModelAndInterface(contextId));
```

Comment:
Ditto here.
Also we prefer iterator over it in WebCore.

Reply:
are you sure?
`auto it = ` -> 986 results in 444 files
`auto iterator = ` -> 304 results in 158 files